sammba/modality_processors/perfusion_fair.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sunday 4th March 2018
+
+@author: Nachiket Nadkarni
+"""
+
+
+import pandas as pd
+import nibabel as nib
+import numpy as np
+from scipy.optimize import least_squares as ls
+from multiprocessing import Pool
+from functools import partial
+import tqdm
+
+
+def perf_fair_read_ptbl(perf_fair_fname):
+    """
+    Extract important information about the perfusion FAIR acquisition

```

Comment:
can you be more explicit about *important* information

Reply:
OK